deploy/crds/integreatly_v1alpha1_gitea_cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Gitea
+metadata:
+  name: example-gitea
+spec:
+  hostname: "gitea.apps.pbraun-dcdc.openshiftworkshop.com"
+  giteaImage: "docker.io/wkulhanek/gitea"
```

Comment:
does it make sense to specify the image here? Why would the operator not know which image to use? I am not sure we want to put that in the CR definition?

Reply:
I thought it would let you newer versions of Gitea without updating the Operator, but that is probably not a good thing: you won't know if a newer version requires different resources, templates, etc. So i will remove that.